client/src/reducers/feedbackReducer.ts

```diff
@@ -0,0 +1,35 @@
+import { Feedback } from 'types';
+import { uid } from 'utils/uid';
+
+export type ActionData = {
+  type: string;
+  payload?: {
+    name: string;
+    data?: string | number | readonly string[] | undefined;
```

Comment:
why `readonly`?

Reply:
something what eslint or ts suggested in vscode :) ... I remove it